lib/rubocop/node_pattern.rb

```diff
@@ -440,11 +442,17 @@ def emit_trailing_params
         params.empty? ? '' : ",#{params}"
       end
 
+      def emit_guard_clause
+        <<-RUBY
+          return nil unless node.is_a?(RuboCop::AST::Node)
```

Comment:
I just created an issue for that: https://github.com/bbatsov/rubocop/issues/4508 :wink: